src/polytopes/chain.jl

```diff
@@ -4,59 +4,31 @@
 
 """
     Chain(p1, p2, ..., pn)
+    Chain(points::AbstractVector)
 
-A polygonal chain from a sequence of points `p1`, `p2`, ..., `pn`.
+A polygonal chain from a sequence of points `p1`, `p2`, ..., `pn`. If `points`
+is of type `CircularVector`, the chain is considered closed.
```

Comment:
```suggestion
A polygonal chain from a sequence of points `p1`, `p2`, ..., `pn`.
If `points` is of type `CircularVector`, the chain is considered closed.
```

Reply:
I’ve made the changes to docstrings more minimal in the newer version – feel free to change further.